src/Text/Heterocephalus.hs

```diff
@@ -522,7 +522,15 @@ docToExp set scope (DocCond conds final) = do
     go (d, docs) = do
       let d' = derefToExp ((specialOrIdent, VarE 'or) : scope) d
       docs' <- docsToExp set scope docs
-      return $ TupE [d', docs']
+      return $ nonUnaryTupE [d', docs']
+      where
+        nonUnaryTupE :: [Exp] -> Exp
+        nonUnaryTupE [e] = e
```

Comment:
It seems like this line would never match, since you're explicitly passing `nonUnaryTupE` two elements.

Reply:
true, we use TupE only once with exact elements :D